.gitignore

```diff
@@ -1,3 +1,10 @@
+csv_data/
```

Comment:
Do NOT change global .gitignore
create .gitignore file in your own directory, "Name Surname/0/.gitignore"